README.md

```diff
@@ -34,39 +34,39 @@ OpenSSH is installed by default, so please change the mobile/root passwords to p
 4. After that, run `passwd` to change its password.
 
 ## Project Sandcastle
-**What's Project Sandcastle:** It's Android and Linux for the iPhone. Projectsandcastle is in beta so not everything is fully supported. [More information](https://projectsandcastle.org)
+**What's Project Sandcastle:** It's Android and Linux for the iPhone. Project Sandcastle is in beta so not everything is fully supported. [More information](https://projectsandcastle.org) and (https://ipadlinux.org)
 ### Installing Android
 Select Project Sandcastle > Setup Project Sandcastle > Start Android.
 When Project Sandcastle has been installed, simply choose "Start Android" to boot back into Android.
 
 **Removing Android:**
-If you wish to remove the Android NAND image and reclaim the space you can login via SSH to your checkra1ned device and mount the final volume and remove the nand file. To do this run `ls /dev/disk0s1s*` and find the last volume. You can verify it's the right volume by running `/System/Library/Filesystems/apfs.fs/apfs.util -p VOLUME_HERE` and if it says Android, that's the correct one. Once you have the volume path you can then run as root (type `su`):
+If you wish to remove the Android NAND image and reclaim the space, you can login via SSH to your Checkra1ned device, and mount the final volume, and remove the NAND file. To do this, run `ls /dev/disk0s1s*` and find the last volume. You can verify it's the right volume by running `/System/Library/Filesystems/apfs.fs/apfs.util -p VOLUME_HERE` and if it says Android, that's the correct one. Once you have the volume path, you can then run these commands as root (type `su`):
```

Comment:
If I'm not wrong, checkra1ned is used as an adjective so it doesn't require a capital letter.